llvm/lib/Support/Unix/Path.inc

```diff
@@ -508,6 +512,9 @@ static bool is_local_impl(struct STATVFS &Vfs) {
 #elif defined(__EMSCRIPTEN__)
   // Emscripten doesn't currently support remote filesystem mounts.
   return true;
+#elif defined(__wasi__)
```

Comment:
`__EMSCRIPTEN__ || __wasi__` is effectively equal `__wasm__`; merge clauses?

Reply:
Does emscripten no longer have asm.js mode? Last time I checked it did, but I'm not sure if it still does today...

Reply:
It doesn't; C++ compilation only targets wasm.

The emscripten driver still supports generating javascript, but only via invoking the binaryen "wasm2js" tool post-link -- which translates the wasm binary into javascript.